src/app/workflow-center/custom-task/custom-task.component.html

```diff
@@ -0,0 +1,130 @@
+<div class='custom-task'>
+
+  <inventory-header [allItems]="allTasks" [name]="'Tasks'"
+  (action)="onAction($event)" (filter)="onFilter($event)">
+  </inventory-header>
+
+  <div class="inventory">
+      <clr-datagrid [clrDgLoading]="dgDataLoading" [(clrDgSelected)]="selectedTasks">
+        <clr-dg-placeholder>{{ dgPlaceholder }}</clr-dg-placeholder>
+        <clr-dg-column [clrDgSortBy]="friendlyNameComparator">
+          <ng-container *clrDgHideableColumn="{hidden: false}">FriendlyName</ng-container>
+          <clr-dg-string-filter [clrDgStringFilter]="friendlyNameFilter"> </clr-dg-string-filter>
+        </clr-dg-column>
+        <clr-dg-column [clrDgSortBy]="injectableNameComparator">
+          <ng-container *clrDgHideableColumn="{hidden: false}">InjectableName</ng-container>
+          <clr-dg-string-filter [clrDgStringFilter]="injectableNameFilter">
+          </clr-dg-string-filter>
+        </clr-dg-column>
+        <clr-dg-column [clrDgSortBy]="implementsTaskComparator">
+            <ng-container *clrDgHideableColumn="{hidden: false}">ImplementsTask</ng-container>
+            <clr-dg-string-filter [clrDgStringFilter]="implementsTaskFilter">
+            </clr-dg-string-filter>
+          </clr-dg-column>
+        <clr-dg-column>
+          <ng-container *clrDgHideableColumn="{hidden: false}">Options</ng-container>
+          <clr-dg-string-filter [clrDgStringFilter]="optionsFilter">
+          </clr-dg-string-filter>
+        </clr-dg-column>
+        <clr-dg-column >
+          <ng-container *clrDgHideableColumn="{hidden: false}">Properties</ng-container>
+          <clr-dg-string-filter [clrDgStringFilter]="propertiesFilter">
+          </clr-dg-string-filter>
+        </clr-dg-column>
+  
+        <clr-dg-row *clrDgItems="let task of tasksStore" [clrDgItem]="task">
+          <clr-dg-action-overflow>
+            <button class="action-item" (click)="onUpdate(task)">Update</button>
+            <button class="action-item" (click)="onDelete(task)">Delete</button>
+            <button class="action-item" (click)="onGetDetails(task)">Detail</button>
+          </clr-dg-action-overflow>
+          <clr-dg-cell>{{task.friendlyName}}</clr-dg-cell>
+          <clr-dg-cell >{{task.injectableName}}</clr-dg-cell>
+          <clr-dg-cell >{{task.implementsTask}}</clr-dg-cell>
+          <clr-dg-cell (click)="getChild('options', task)">
+            <u *ngIf="task.options  && (task.options | json) != '{}'" class="grid-item">Options</u>
```

Comment:
hummm this comparison is strange

Reply:
The options (and properties) can arribe as empty json. In this case, the link should not be shown since they don't have anything to show. I changed it to only `(task.options | json) != '{}'`